### PR TITLE
updates for python 3

### PR DESCRIPTION
**Pull request by kaalleen**

### CODE REVIEW

File: pyembroidery-convert.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
-from __future__ import print_function
```
Comment (lexelby) [inline pyembroidery-convert.py:1]: I vote we keep these here for compatibility with python 2.7.  What's the harm?

File: pyembroidery/StringHelper.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 def is_string(thing):
     try:
-        return isinstance(thing, basestring)
```
Comment (lexelby) [inline pyembroidery/StringHelper.py:3]: I'd like to keep compatibility with python 2.7 if we can, to aid in the transition.  Maybe a try/except here to catch a NameError for basestring?

```
try:
    return isinstance(thing, basestring)
except NameError:
    return isinstance(thing, str)
```

**Comment by kaalleen** [comment]
Sure, why not keep compatibility. I just ran the 2to3 command on this. As long as it works with the new versions I am ok with it.

**Review by lexelby — APPROVED** [review]

**Comment by lexelby** [comment]
Sorry, totally forgot to approve this!

**Pull request merged**